From bda81698b64f2ca44535d5cd7eb5e138de7dde4e Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 17 Jun 2015 07:50:16 +0200
Subject: [PATCH 01/11] Disable Educational device

Message-id: <e4fa7c923e8b99992be5f9ec7d5b76edb63ad0db.1434439667.git.mrezanin@redhat.com>
Patchwork-id: 66275
O-Subject: [RHEV-7.2 qemu-kvm-rhev PATCH 1/3] Disable Educational device
Bugzilla: 1194151
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

Rebase to qemu-2.3.0 introduce edu device - dummy educational device.
This device is not supported by libvirt and we are not going to
support it with RHEL.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/pci.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index 16d1700..460e48a 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -18,7 +18,6 @@ CONFIG_WDT_IB6300ESB=y
 CONFIG_PCI_TESTDEV=y
 CONFIG_SD=y
 CONFIG_SDHCI=y
-CONFIG_EDU=y
 CONFIG_VGA=y
 CONFIG_VGA_PCI=y
 CONFIG_IVSHMEM=$(CONFIG_KVM)
-- 
1.8.3.1

